src/main/java/com/exercises/first/DuplicateFinder.java

```diff
@@ -0,0 +1,26 @@
+package com.exercises.first;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DuplicateFinder {
+    public static <T> List<T> findDuplicates(List<T> list) {
+        Map<T, Integer> temp = new LinkedHashMap<>();
```

Comment:
Why wasn't it resolved by LinkedHashSet? It will do almost the same but easier ;)